gravitee-apim-plugin/gravitee-apim-plugin-entrypoint/gravitee-apim-plugin-entrypoint-webhook/src/main/java/io/gravitee/plugin/entrypoint/webhook/WebhookEntrypointConnector.java

```diff
@@ -159,51 +181,33 @@ private Completable sendAndDiscard(String requestUri, HttpClient httpClient, Mes
             .onErrorComplete();
     }
 
-    private Completable prepareClientOptions(final ExecutionContext ctx) {
+    private Completable prepareExecutionContext(final ExecutionContext ctx) {
```

Comment:
maybe be more precise ?
```suggestion
    private Completable computeSubscriptionContextAttributes(final ExecutionContext ctx) {
```

Reply:
:heavy_check_mark: